papers/marlowe2019/paper.tex

```diff
@@ -510,27 +508,23 @@ \subsection{Valid state and positive account preservation}
 
 \subsubsection{Quiescent result}
 
-A contract is \emph{quiescent} if and only if the root construct \texttt{When}, or if the contract is \texttt{Close} and all accounts are empty.
-
-We have proved that, if an input \texttt{State} is valid and all accounts are positive, then the result from processing any transaction will always be quiescent.
+A contract is \emph{quiescent} if and only if the root construct \texttt{When}, or if the contract is \texttt{Close} and all accounts are empty. We have proved that, if an input \texttt{State} is valid and accounts are positive, then the output will be quiescent.
 
 \subsection{Money preservation and contract timeout}
 
 One of the dangers of using smart contracts is that a badly written one can potentially lock its funds forever. By the end of the contract, all the money paid to the contract must be distributed back, in some way, to a subset of the participants of the contract.
 To ensure this is the case we proved two properties:
 
 \subsubsection{Money preservation}
-Money is not created or destroyed by the semantics. More specifically, the money that comes in plust the money in the contract before the transaction must be equal to the money that comes out plus the contract after the transaction, except in the case of an error, when the contract stays unaltered.
+Money is not created or destroyed by the semantics. More specifically, the money that comes in plus the money in the contract before the transaction must be equal to the money that comes out plus the contract after the transaction, except in the case of an error.
 
 \subsubsection{Timeout closes a contract}
 
 For every Marlowe \texttt{Contract} there is a slot number after which an empty transaction can be issued that will close the contract and refund all the money in its accounts.
 
 A conservative upper bound for the expiration slot number can be calculated efficiently by using the function \texttt{maxTime} (or \texttt{maxTimeContract} in the Isabelle semantics), essentially by taking the maximum of all the timeouts in the contract.
 
-We proved that this conservative upper bound is general enough for every contract, by showing that, if the contract is not closed and empty, then an empty transaction sent after \texttt{maxTime} will close the contract and leave the accounts empty.
-
-Because of this property and \emph{Money preservation}, we can conclude that money will not be locked in the contract forever.
```

Comment:
is this not an important point?

Reply:
Yes, but we basically already say it in lines 515 and 516